### DISCUSSION TITLE
请问编译后的pdf文档，放在知网查重中，查重结果都是乱码是为什么。因为中文都是乱码所以查重率0.01%

### DISCUSSION THREAD

**Discussion by ppppls115** [note286/xduts#179] [Q&A] [answered]
不知道是什么情况

**Comment by note286** [comment, marked as answer]
可以尝试复制PDF中的文本，并粘贴至记事本中，如果正确显示文本，那么说明正常。

**Reply by dodo-m** [reply]
我发现我用知网个人版查重，公式在查重报告里显示了，正常PDF是应该识别公式吗？因为word查重不识别公式，我提交的PDF（因为原始PDF过大，我用福昕编辑器压缩了一下）可以识别公式，这样虽然识别公式但是查不了重，是不是不准呢？学校的系统是怎样的呢？

**Reply by note286** [reply]
公式也是属于文本的范畴，是可以搜索和复制的，被查重系统识别出来是正常现象。

你是说有部分公式被标记为重复吗？

**Reply by dodo-m** [reply]
我的意思是，公式没有标为重复，但算字符数，这样重复率就会偏低，不知道学校系统查出来会是怎样的

**Reply by note286** [reply]
学校的也会识别出公式的字符，只是有点乱。

**Reply by dodo-m** [reply]
那应该和知网个人版查重差不多，感谢解答！！
